src/linsolve.jl

```diff
@@ -1,21 +1,15 @@
 const PFac = PositiveFactorizations
 
-# A non-allocating version of ipiv2perm from LinearAlgebra/src/lu.jl
-function ipiv2perm!(p::AbstractVector{T}, v::AbstractVector{T}, maxi::Integer) where T
-    length(p) == maxi || throw(DimensionMismatch("length of p must be $maxi"))
-    @inbounds for i in 1:maxi
-        p[i] = i
-    end
+function apply_ipiv_perm!(w::AbstractVector, v::AbstractVector{<:Integer})
```

Comment:
There used to be a third argument indicating the length of the permutation vector, which shouldn't matter in the typical use case where the preallocation is made correctly. The CI all failed because that third argument is gone.